### PR TITLE
[SCHEDULED for Nov 24, do not merge before] feat(blog): hoodie updates

### PR DESCRIPTION
**Pull request by janl**
Part 2 in our little series.

### CODE REVIEW

File: _posts/2015-11-24-hoodie-updates.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+This work is already available in our new @next tag on npm, albeit not 100% functional yet. If you are keen on giving it a spin and help fix missing features, run *npm install hoodie@next* to get the work-in-progress version.
+
+See https://github.com/hoodiehq/hoodie for more details and a a way to find and file issues. The main work is done in the [Client](https://github.com/hoodiehq/hoodie-client), [Server](https://github.com/hoodiehq/hoodie-server) and [Admin Dashboard](https://github.com/hoodiehq/hoodie-admin-dashboard) modules.
+
+We’ve re-written the Admin Dashboard in Ember and we are very pleased with the results.
+
+In a touch decision between shipping something that works vs. all the features we would like, we have decided to delay the introduction of plugins for the Hoodie 1.0 beta. Plugins will be back before long, but in the current state of refactoring, they are slowing us down, and so [we are cutting them for now](https://github.com/hoodiehq/hoodie-server/issues/417), only to re-introduce them at a more opportune time.
```
Comment (garrensmith) [inline _posts/2015-11-24-hoodie-updates.md]: s/touch/tough

File: _posts/2015-11-24-hoodie-updates.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: "Hoodie 1.0 milestones closing in"
```
Comment (varjmes) [inline _posts/2015-11-24-hoodie-updates.md]: The title doesn't read well to me. "Hoodie 1.0 Milestones: Closing In" appears better to my eyes, or something like that?

File: _posts/2015-11-24-hoodie-updates.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+author: Jan
+layout: blog-post-tgif
```
Comment (varjmes) [inline _posts/2015-11-24-hoodie-updates.md]: Why TGIF and not a standard layout? Should we make TGIF our default? :P

File: _posts/2015-11-24-hoodie-updates.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+comments: true
+---
+
+This is Part 2 of of our [little series about what has happened in Hoodie-land](http://hood.ie/blog/we-have-been-busy.html "We. Have. Been. Busy!") in the past <strike>three</strike>five months.
+
+Big news, big news: we’re getting there! After [a very busy summer](prev blog post) we are closing in on the final tasks for [the Hoodie 1.0 milestones](http://gr2m.github.io/milestones/).
```
Comment (varjmes) [inline _posts/2015-11-24-hoodie-updates.md]: Link to prev blog post to insert on this line: http://hood.ie/blog/we-have-been-busy.html

File: _posts/2015-11-24-hoodie-updates.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+Most notably, we [completely re-structured how Hoodie is put together, and tested and released](https://github.com/hoodiehq/discussion/issues/76).
```
Comment (varjmes) [inline _posts/2015-11-24-hoodie-updates.md]: "completely re-structured how Hoodie is put together, tested and released"

File: _posts/2015-11-24-hoodie-updates.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+This work is already available in our new @next tag on npm, albeit not 100% functional yet. If you are keen on giving it a spin and help fix missing features, run *npm install hoodie@next* to get the work-in-progress version.
```
Comment (varjmes) [inline _posts/2015-11-24-hoodie-updates.md:14]: Link to @next tag? :)

File: _posts/2015-11-24-hoodie-updates.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+See https://github.com/hoodiehq/hoodie for more details and a a way to find and file issues. The main work is done in the [Client](https://github.com/hoodiehq/hoodie-client), [Server](https://github.com/hoodiehq/hoodie-server) and [Admin Dashboard](https://github.com/hoodiehq/hoodie-admin-dashboard) modules.
```
Comment (varjmes) [inline _posts/2015-11-24-hoodie-updates.md]: Will this first link be rendered clickable when published?

Comment (varjmes) [inline _posts/2015-11-24-hoodie-updates.md]: remove the second "a". "a a way to find" -> "a way to find"

File: _posts/2015-11-24-hoodie-updates.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
+---
+title: "Hoodie 1.0 milestones closing in"
+author: Jan
+layout: blog-post-tgif
+comments: true
+---
+
+This is Part 2 of of our [little series about what has happened in Hoodie-land](http://hood.ie/blog/we-have-been-busy.html "We. Have. Been. Busy!") in the past <strike>three</strike>five months.
```
Comment (gr2m) [inline _posts/2015-11-24-hoodie-updates.md:8]: I’d say "in the past five months" and leave the "three" out entirely. 

But I don't mind, so if you like it better that way, please leave it in :)

**Comment by gr2m** [comment]
:+1: Great comments, too :+1:

File: _posts/2015-11-24-hoodie-updates.md (hunk 2 above)
Comment (janl) [inline _posts/2015-11-24-hoodie-updates.md]: good one, wen’t with the “Hoodie 1.0 Milestones: Closing In” variant

File: _posts/2015-11-24-hoodie-updates.md (hunk 3 above)
Comment (janl) [inline _posts/2015-11-24-hoodie-updates.md]: actually, there is a tgif-ish layout for this, fixed in next push.

File: _posts/2015-11-24-hoodie-updates.md (hunk 6 above)
Comment (janl) [inline _posts/2015-11-24-hoodie-updates.md:14]: there is no link to a tag on npm :/

File: _posts/2015-11-24-hoodie-updates.md (hunk 8 above)
Comment (janl) [inline _posts/2015-11-24-hoodie-updates.md:8]: it’s part of the joke that we are a little late :)

**Comment by janl** [comment]
Thanks all! <3

**Pull request merged**